core/util/src/main/scala/net/liftweb/util/Props.scala

```diff
@@ -53,17 +54,45 @@ object Props extends Logger {
    * @param name key for the property to get
    * @return the value of the property if defined
    */
-  def get(name: String): Box[String] = Box(props.get(name))
+  def get(name: String): Box[String] = {
+    val tries:List[Box[Object]] =
+      prepended.map(m => Box.legacyNullTest(m get name)) ++
+      (props.get(name).toBox ::
+      appended.map(m => Box.legacyNullTest(m get name)))
+
+    tries.find(_.isDefined).getOrElse(Empty).map(interpolate)
+  }
+
+  private val INTERPOLATE = """(.*?)\Q${\E(.*?)\Q}\E([^$]*)""".r
+  private def interpolate(v:Object):String = {
```

Comment:
:-1: on naming these two the same thing. Let's name the all-caps `INTERPOLATE` to `interpolateRegex` or something like that.